### ISSUE TITLE
Error when compile many projects in solution

### ISSUE THREAD

**Issue by colderlord** [bridgedotnet/Bridge#3824]
I did some [enhancement ](https://github.com/bridgedotnet/Bridge/issues/3805) and broke something
Error:
``` 
Error		Object reference not set to an instance of an object.    at Bridge.Translator.Emitter.GetTypeDefinition(AstType reference, Boolean safe)
   at Bridge.Translator.ObjectCreateBlock.VisitObjectCreateExpression()
   at Bridge.Translator.ObjectCreateBlock.EmitConversionExpression()
   at Bridge.Translator.ConversionBlock.DoEmit()
   at Bridge.Translator.AbstractEmitterBlock.Emit()
   at Bridge.Translator.Emitter.VisitObjectCreateExpression(ObjectCreateExpression objectCreateExpression)
   at ICSharpCode.NRefactory.CSharp.ObjectCreateExpression.AcceptVisitor(IAstVisitor visitor)
   at Bridge.Translator.VariableBlock.VisitVariableDeclarationStatement()
   at Bridge.Translator.VariableBlock.DoEmit()
   at Bridge.Translator.AbstractEmitterBlock.Emit()
   at Bridge.Translator.Emitter.VisitVariableDeclarationStatement(VariableDeclarationStatement variableDeclarationStatement)
   at ICSharpCode.NRefactory.CSharp.VariableDeclarationStatement.AcceptVisitor(IAstVisitor visitor)
   at Bridge.Contract.Helpers.AcceptChildren(AstNode node, IAstVisitor visitor)
   at Bridge.Translator.ConstructorBlock.EmitCtorForInstantiableClass()
   at Bridge.Translator.ConstructorBlock.DoEmit()
   at Bridge.Translator.AbstractEmitterBlock.Emit()
   at Bridge.Translator.ClassBlock.EmitInstantiableBlock()
   at Bridge.Translator.ClassBlock.DoEmit()
   at Bridge.Translator.AbstractEmitterBlock.Emit()
   at Bridge.Translator.EmitBlock.DoEmit()
   at Bridge.Translator.AbstractEmitterBlock.Emit()
   at Bridge.Translator.Emitter.Emit()
   at Bridge.Translator.Translator.Translate()
   at Bridge.Build.BridgeCompilerTask.Execute()
```

I will fix this soon as possible. I'm sorry Guys :-[

**Comment by BojidarStanchev** [comment]
I get this error and I have no idea why? Any pointers on how to resolve it?

**Comment by colderlord** [comment]
When Bridge team will approve my PR, it will be fixed

**Comment by geoffreymcgill** [comment]
I have tagged this issue and related PR to be reviewed and hopefully included in the 17.7.0 release.

**Comment by fabriciomurta** [comment]
The fix has just been accepted and merged in, it will be available next Bridge release (see attached milestone).
